rosdep/python.yaml

```diff
@@ -122,6 +122,19 @@ gunicorn:
   fedora: [python-gunicorn]
   gentoo: [www-servers/gunicorn]
   ubuntu: [gunicorn]
+imgaug-pip:
```

Comment:
@ros/rosdeputies I'm not sure if we have strong feelings about whether the key name should be prefixed with python or not.

Reply:
Looking at [the current status](https://github.com/ros/rosdistro/blob/c6195496facec04d23a7b98d2bbe29f9ab66482b/rosdep/python.yaml) of pkgs with `*-pip` suffix, some are prefixed by `python-` while others aren't. IMO `-pip` makes it clear enough that a package is Python and taken from pip, unless there's a non-pip package that ends with -pip.

Reply:
No strong feeling here.

Reply:
I have my thoughts updated in https://github.com/ros/rosdistro/pull/21239#issuecomment-492866362